Components/Alerts/alert.css

```diff
@@ -0,0 +1,64 @@
+
+.alert {
+  box-sizing: border-box;
+  width: 100%;
+  border-radius: 5px;
+  padding: 1.25rem 2.5rem;
+  box-shadow: 0 0 15px 5px #ccc;
+  border-left: 0.5rem solid;
+}
+.alert-text {
+  padding: 0 1rem;
+}
+
+.alert-icon {
+  position: absolute;
+  left: 20px;
+  top: 50%;
+  transform: translateY(-50%);
+  font-size: 1.5rem;
+}
+.close-btn {
+  position: absolute;
+  right: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  padding: 1rem 1rem;
+  cursor: pointer;
+}
+
+.close-btn .fa-times {
+  font-size: 1.375rem;
+  line-height: 2.5rem;
+}
+
+.alert-warning {
+  background-color: #fff3cd;
+  position: relative;
+  border-left: 0.5rem solid #ffd48a;
+  color: #856404;
+}
```

Comment:
use css variable for color so it will be convenient when ever you wish to change color scheme of a project where you would be using your component library
Refer to design system provided by tanay